composer.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "lmc/matej",
```

Comment:
neni to zbytecne konkretni? co kdyz budou dalsi knihovny tykajici se mateje? nebylo by lepsi napr `lmc/matej-api-client`

Reply:
Jako jake napriklad knihovny? Zase mi prijde zbytecny delat to specificky, kdyz zadna takova knihovna nejspis nikdy nebude :). A bundle nazveme jinak.

Reply:
jo muze byt.. jen jsem na to chtel upozornit, jestli nekdo treba nevi o necem uz ted.. protoze tohle je neco co se v budoucnu dost blbe meni

Reply:
a tim, ze tomu das konkretnejsi nazev taky zarucis i SRP

Reply:
Uvidime, promyslel jsem to, ale zase jsem spise pro jednoduchost, nez tomu ted davat nejaky premature specificky nazev :).